Gems/Atom/RHI/Vulkan/CMakeLists.txt

```diff
@@ -13,3 +13,5 @@ o3de_restricted_path(${gem_json} gem_restricted_path gem_parent_relative_path)
 ly_add_external_target_path(${CMAKE_CURRENT_LIST_DIR}/3rdParty)
 
 add_subdirectory(Code)
+
+ly_create_alias(NAME Atom_RHI_Vulkan.HeadlessServers NAMESPACE Gem)
```

Comment:
This should be added in the Vulkan/**Code**/CMakeLists.txt next to the other aliases